src/Desktop.Analyzers/Desktop.Analyzers.md

```diff
@@ -48,6 +48,16 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182350.aspx](http://msdn.microsoft.com/library/ms182350.aspx)
```

Comment:
I think we should use the Markdown syntax for delimiting code fragments, \`XmlReader\` for `XmlReader`. Thoughts?